src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -435,9 +453,11 @@ TEST_F(ResumeCtrlTest, StartResumption_AppWithSubscriptionToIVI) {
       test_subscriptions;
 
   // Check RestoreApplicationData
+  EXPECT_CALL(app_mngr_, GetDefaultHmiLevel(_))
+      .WillRepeatedly(Return(default_test_level));
   GetInfoFromApp();
   EXPECT_CALL(*mock_storage, GetSavedApplication(_, _, _))
-      .Times(3)
+      .Times(2)
```

Comment:
Number of `GetSavedApplication(....)` calls stays the same from test to test. Maybe be better to replays it with constant?